src/utils/string/index.ts

```diff
@@ -110,14 +110,19 @@ export const omitTypename = (object) =>
     key === "__typename" ? undefined : value
   );
 
-export const getDateCopy = (time: Date, tz?: string) => {
+export const getDateCopy = (time: Date, tz?: string, dateOnly?: boolean) => {
   if (tz) {
-    return formatToTimeZone(time, "MMM D, YYYY h:mm:ss a", {
+    const dateFormat = dateOnly ? "MMM D, YYYY" : "MMM D, YYYY h:mm:ss a";
```

Comment:
we should use a lowercase "d" since `D` represents day of year, where as `d` represents the day of month  
https://date-fns.org/v2.16.1/docs/format